apple-pie/index.html

```diff
@@ -3,37 +3,53 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/styles/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="styles/style.css">
```
Acuerdate que si le especificas la / le estas diciendo a la pagina que se vaya a la raiz del ordenador para buscar esa carpeta, styles no esta ahi esta en la carpeta apple-pie, que es de donde abrimos el Go Live, puedes ponerle la /, pero necesitas especificarle el ./, para decirle que es dentro de esta carpeta tengo styles